tools/sql_extraction/src/main/kotlin/DataFlowSolver.kt

```diff
@@ -29,6 +29,7 @@ class DataFlowSolver(private val frontends: List<FrontEnd>) {
             }
         }
 
-        throw IllegalArgumentException("Cannot analyze $filePath. No frontend available.")
+        LOGGER.warn { "Cannot analyze $filePath. No frontend available. Skipping file..." }
```

Comment:
nit: should you print the filePath somewhere? $filePath doesn't have much information or it would be replaced by the actual variable?

Reply:
$ is string interpolation in Kotlin. It should be replaced by the actual variable's toString().

Reply:
thanks!